extensions/interactions/SetInput/SetInput.py

```diff
@@ -22,9 +22,11 @@ class SetInput(base.BaseInteraction):
 
     name = 'Set Input'
     description = 'Allows learners to enter an unordered set of strings.'
-    display_mode = base.DISPLAY_MODE_SUPPLEMENTAL
+    display_mode = base.DISPLAY_MODE_INLINE
```

Comment:
I think 'inline' here means 'in the flow', as opposed to 'fits in one line'. I think this is the usual meaning of the phrase 'inline image'.

Having said that, I'm open to other suggestions for the name. I don't have a better one at the moment, though.